path_login.go

```diff
@@ -103,8 +103,11 @@ func (b *kubeAuthBackend) pathLogin(ctx context.Context, req *logical.Request, d
 		return nil, err
 	}
 
+	b.l.RLock()
```

Comment:
Don't we already have the read lock from line 69 above?

Reply:
You are right! However, I don't believe we would run into a deadlock I could be mistaken.